app/models/cloud_volume_snapshot.rb

```diff
@@ -26,6 +26,61 @@ def my_zone
     self.class.my_zone(ext_management_system)
   end
 
+  def self.create_snapshot_queue(userid, cloud_volume, options = {})
+    ext_management_system = cloud_volume.&ext_management_system
```

Comment:
Think this is supposed to be
```suggestion
    ext_management_system = cloud_volume&.ext_management_system
```

We might want to raise some errors if `cloud_volume` or `cloud_volume.ext_management_system` are nil though instead of just safe navigating around it.  E.g. this line might pass but `cloud_volume.class.name` and `ext_management_system.queue_name_for_ems_operations` will blow up below.

Thoughts on `raise ArgumentError if cloud_volume.nil? || cloud_volume.ext_management_system.nil?`

Reply:
@kbrock it looks like this is why your cross-repo test is failing: https://github.com/ManageIQ/manageiq-cross_repo-tests/runs/5837952177?check_suite_focus=true#step:6:2209

`"message"=>"undefined method '&' for #<ManageIQ::Providers::Openstack::StorageManager::CinderManager::CloudVolume:0x0000561314985e40>"`